projects/common/lib/components/name/name.component.html

```diff
@@ -0,0 +1,67 @@
+
+<div class="row">
+    <div class="col-md-4">
+        <div class="form-group"
+              [ngClass]="{'has-error': showError && (firstNameRef.touched || firstName === undefined || firstName === null)}">
+            <label class="control-label" for="firstName{{objectId}}"
+                    >{{firstNamelabel}}</label>
+            <input type="text"
+                    [(ngModel)]="firstName"
```

Comment:
You can just do `[ngModel]` here instead of `[(ngModel)]`.  Why? Because you set the model value in `setFirstName($event)` by doing `this.person.firstName = value;`

Right now you're doubling up on writing the value for no reason.